sandpack-react/src/components/Preview/Preview.stories.tsx

```diff
@@ -122,3 +122,14 @@ export const AdditionalButtons: React.FC = () => (
     </SandpackLayout>
   </SandpackProvider>
 );
+
+export const AdditionalContent: React.FC = () => (
```

Comment:
Awesome, I appreciate your effort to keep the documentation and storybook up to date!